src/olympia/amo/log.py

```diff
@@ -671,6 +671,21 @@ class REVIEWER_REPLY_VERSION(_LOG):
     review_queue = True
 
 
+class APPROVALNOTES_CHANGED(_LOG):
```

Comment:
you might want to add this to https://github.com/mozilla/addons-server/blob/master/src/olympia/activity/utils.py#L228 - those are the events that make notes before them 'read'.